core/src/main/java/data/movies/datasource/localstore/CreditsStore.kt

```diff
@@ -11,14 +11,14 @@ import javax.inject.Singleton
 @Singleton
 class CreditsStore @Inject constructor() {
     // Map<movieId, actors>
-    private val _movieActors = MutableSharedFlow<Map<Int, List<Actor>>>(replay = 1)
+    private val _movieActors = MutableSharedFlow<Map<Int, List<ActorPart>>>(replay = 1)
 
     @OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
@OptIn(ExperimentalCoroutinesApi::class) is redundant

Reply:
ok